l1-contracts/test/foundry/unit/concrete/state-transition/chain-deps/facets/Mailbox/TransferEthToSharedBridge.t.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {MailboxTest} from "./_Mailbox_Shared.t.sol";
+import {IL1SharedBridge} from "contracts/bridge/interfaces/IL1SharedBridge.sol";
+
+contract MailboxTransferEthToSharedBridge is MailboxTest {
+    address baseTokenBridge;
+
+    function setUp() public virtual {
+        prepare();
+
+        baseTokenBridge = makeAddr("bridge");
+        vm.deal(diamondProxy, 1 ether);
+        utilsFacet.util_setChainId(eraChainId);
+        utilsFacet.util_setBaseTokenBridge(baseTokenBridge);
+    }
+
+    modifier useBaseTokenBridge() {
+        vm.startPrank(baseTokenBridge);
+        _;
+        vm.stopPrank();
+    }
+
+    function test_success_transfer() public useBaseTokenBridge {
+        vm.mockCall(baseTokenBridge, abi.encodeWithSelector(IL1SharedBridge.receiveEth.selector, eraChainId), "");
```

Comment:
Why is there a need to do a `mockCall`? 

We are calling the `IL1SharedBridge.receiveEth` function by calling `mailboxFacet.transferEthToSharedBridge`. We are calling the mailbox function while mocking the calling address as the `baseTokenBridge`, which is pretty much the only condition for the function `receiveEth` to work properly. Therefore, ideally, we won't need the `mockCall`.

Please re-write this without using `mockCall` AND as a general rule of thumb, let's minimize the usage of `mockCall` to the absolute minimum.

Reply:
Fixed, replaced with a call to dummy bridge.